test/ContainerCommandLoaderTest.php

```diff
@@ -10,7 +10,7 @@
 
 namespace LaminasTest\Cli;
 
-use Laminas\Cli\ContainerCommandLoader;
+use laminas\cli\containercommandloader;
```

Comment:
Please revert (all changes in that file), I would suggest adding the sniff rule to be ignored in the repo. It doesn't work correctly with class aliases (PHP store class aliases always as lowercase)